exporter/elasticsearchexporter/config_test.go

```diff
@@ -360,6 +361,22 @@ func TestConfig(t *testing.T) {
 				cfg.MetadataKeys = []string{"x-test-1", "x-test-2"}
 			}),
 		},
+		{
+			id:         component.NewIDWithName(metadata.Type, "queuebatch_enabled"),
+			configFile: "config.yaml",
+			expected: withDefaultConfig(func(cfg *Config) {
+				cfg.Endpoint = "https://elastic.example.com:9200"
+
+				cfg.QueueBatchConfig.Enabled = true
+				cfg.QueueBatchConfig.NumConsumers = 100
+				cfg.QueueBatchConfig.Sizer = exporterhelper.RequestSizerTypeRequests
+				cfg.QueueBatchConfig.Batch = configoptional.Some(
+					exporterhelper.BatchConfig{
+						Sizer: exporterhelper.RequestSizerTypeRequests,
+					},
+				)
+			}),
+		},
 	}
```

Comment:
Should there be a test case ensuring that the `QueueBatchConfig` takes precendence over the `Batch` config?